src/components/dialogs/export-dialog.jsx

```diff
@@ -54,9 +60,31 @@ const ExportDialog = ({
     const [selectedFormat, setSelectedFormat] = React.useState('');
     const [loading, setLoading] = React.useState(false);
     const [exportStudyErr, setExportStudyErr] = React.useState('');
+    const [studyName, setStudyName] = useState(null);
+    const { snackError } = useSnackMessage();
 
     const [unfolded, setUnfolded] = React.useState(false);
 
+    const fetchStudyName = useCallback(() => {
+        fetchDirectoryElementPath(studyUuid)
+            .then((response) => {
+                const studyName = response[response.length - 1]?.elementName;
+                setStudyName(studyName);
+            })
+            .catch((error) => {
+                snackError({
```

Comment:
Need to test the error

Reply:
tested 
![fail](https://github.com/user-attachments/assets/3c65a3a2-0302-420d-b5f9-67e08cf9d04f)